src/sagas/networkSettings.js

```diff
@@ -3,10 +3,22 @@ import { config } from '@hathor/wallet-lib';
 import { isEmpty } from 'lodash';
 import AsyncStorage from '@react-native-async-storage/async-storage';
 import { featureToggleUpdate, networkSettingsUpdateFailure, networkSettingsUpdateSuccess, reloadWalletRequested, types } from '../actions';
-import { HTTP_REQUEST_TIMEOUT, NETWORK, networkSettingsKey, NETWORK_TESTNET, STAGE, STAGE_DEV_PRIVNET, STAGE_TESTNET } from '../constants';
+import { HTTP_REQUEST_TIMEOUT, NETWORK, networkSettingsKeyMap, NETWORK_TESTNET, STAGE, STAGE_DEV_PRIVNET, STAGE_TESTNET } from '../constants';
 import { getFullnodeNetwork, getWalletServiceNetwork } from './helpers';
 import { STORE } from '../store';
 
+/**
+ * Initialize network settings saga.
+ *
+ * It looks up a stored network settings to upddate the redux state.
+ */
+export function* initNetworkSettings() {
+  const networkSettings = STORE.getItem(networkSettingsKeyMap.networkSettings);
+  if (networkSettings) {
+    yield put(networkSettingsUpdateSuccess(customNetwork));
```

Comment:
Where is `customNetwork` defined?

Reply:
- fix declaration: 24166c7d1e68f4b5e3340c8c4ce919d3293b87c5